### DISCUSSION TITLE
Improve endpoints section in DevUI

### DISCUSSION THREAD

**Discussion by lordofthejars** [quarkusio/quarkus#43445] [Community]
Hi, I have been working on an example of Quarkus where an endpoint uses the POST HTTP method. In DevUI, there's the Endpoints section, but at this point, it only shows the link, so if I click, it invokes the browser, returning an error.

Then, one of the things I have in this project is the OpenAPI dependency set, so I thought if it wouldn't be a good idea if there is OpenAPI (and Swagger UI) deployed when you click this endpoint, it just redirects to Swagger UI.

**Comment by phillip-kruger** [comment]
Should be doable. I'll have a look

**Comment by phillip-kruger** [comment]
@lordofthejars #43950 should fix this.

**Reply by lordofthejars** [reply]
oh great, thanks.
